### PR TITLE
Challenge Administration

### PR DESCRIPTION
**Pull request by thomasyu888**
Challenge documentation

### CODE REVIEW

**Comment by thomasyu888** [comment]
https://github.com/Sage-Bionetworks/synapseDocs/issues/157

**Comment by thomasyu888** [comment]
This is a big pull request.  I'm not sure if we should have all of this in one page.

**Comment by thomasyu888** [comment]
I still have to add another section for challenge admin functions

File: articles/challenge.md [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+	{% tab Web %}
+Navigate to an uploaded file in synapse and click on tools and click submit to challenge
+<img src="/assets/images/howtosubmit.png">
+After doing so, pick the challenge you want to submit to, in this case (My Example Challenge). Click Next and follow the steps to complete your submission.
+<img src="/assets/images/submitToChallenge.png">
+	{% endtab %}
+
+{% endtabs %}
+
+
```
Comment (larssono) [inline articles/challenge.md]: Move to separate article in how-tos.

File: articles/challenge_administration.md [hunk 2]
```diff
@@ -0,0 +1,496 @@
+---
+title: Running a Challenge
+layout: article
+excerpt: Learn how to administer and start a challenge
```
Comment (kdaily) [inline articles/challenge_administration.md]: Learn how to start and administer a Challenge in Synapse

**Pull request merged**